distributed/shuffle/tests/test_shuffle_extension.py

```diff
@@ -15,11 +18,22 @@
 
 
 @gen_cluster([("", 1)])
-async def test_installation(s, a):
+async def test_installation_on_worker(s, a):
     ext = a.extensions["shuffle"]
     assert isinstance(ext, ShuffleWorkerExtension)
     assert a.handlers["shuffle_receive"] == ext.shuffle_receive
     assert a.handlers["shuffle_inputs_done"] == ext.shuffle_inputs_done
+    assert a.stream_handlers["shuffle-fail"] == ext.shuffle_fail
+
+
+@gen_cluster([("", 1)])
+async def test_installation_on_scheduler(s, a):
+    ext = s.extensions["shuffle"]
+    assert isinstance(ext, ShuffleSchedulerExtension)
+    assert s.handlers["shuffle_get"] == ext.get
+    assert (
+        s.handlers["shuffle_get_participating_workers"] == ext.get_participating_workers
+    )
```

Comment:
Curious if these truly provide value. I don't mind having them but they seem to not offer a lot of signal, do they?

Reply:
`test_installation` had been in a half-baked state before. I'm completely fine with removing these, but while we have something like `test_installation`, it should be complete.

Reply:
I'm fine either way.